### PR TITLE
fix whitespace LM tokenize issue

### PR DESCRIPTION
**Pull request by howl-anderson**
**Proposed changes**:
- Skip token when its text can't be tokenized by LM tokenizer (for example, whitespace)

fixs #7406

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by sara-tagger** [comment]
Thanks for submitting a pull request 🚀 @rctatman will take a look at it as soon as possible ✨

**Review by tmbo — CHANGES_REQUESTED** [review]
looks good :+1: can you please add a test to the unit tests in tests/nlu/featurizers/test_lm_featurizer.py as well as a changelog file to changelog/?

**Comment by howl-anderson** [comment]
@tmbo Sure! I will add those features in later commits.

**Comment by howl-anderson** [comment]
@tmbo Please take a look

File: changelog/7407.bugfix.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Remove token when its text (for example, whitespace) can't be tokenized by LM tokenizer (from `LanguageModelFeaturizer`)
```
Comment (tmbo) [inline changelog/7407.bugfix.md:1]:
```suggestion
Remove token when its text (for example, whitespace) can't be tokenized by LM tokenizer (from `LanguageModelFeaturizer`).
```

**Review by tmbo — APPROVED** [review]
thanks a lot for this whitespace fix - really cool :rocket:

**Pull request merged**